Classes/Domain/Repository/ConfigurationRepository.php

```diff
@@ -68,9 +73,14 @@ public function findAll()
      */
     public function findByUid($uid)
     {
-        $where = 'uid=' . (int)$uid . $this->getWhereClauseForEnabledFields();
+        $config = unserialize($GLOBALS['TYPO3_CONF_VARS']['EXT']['extConf']['ig_ldap_sso_auth']);
```

Comment:
Here as well. But why unserializing each time? Better would be to have it once for all in the constructor.